src/styles/custom-antd.css

```diff
@@ -23990,7 +23990,7 @@ tr.ant-table-expanded-row .ant-descriptions-view table {
   min-height: 32px;
   padding: 6px 8px;
   color: #fff;
-  text-align: left;
+  text-align: center;
```

Comment:
what is this for? 
cant see enough of the file to determine

Reply:
the tooltip in compare charts had the text left-aligned. I aligned the text in the tooltip to the center so it would look nicer.

Reply:
In that case, any reason why its also custom set here: https://github.com/HeartUoA/HeaRT/pull/64/files#diff-bc5523715d0652cc86446d40c53f95a9R67

Reply:
whoopsie, forgot I left it in there, will remove now!